Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py

```diff
@@ -45,7 +45,7 @@ def get_incident_init_params():
         'threat_level': demisto.params().get('threat_level', None),
         'threat_type': demisto.params().get('threat_type', None)
     }
-    return {param_k: param_v for param_k, param_v in params_dict.items() if param_v is not None}
+    return {param_k: param_v for param_k, param_v in params_dict.items() if param_v is not None and param_v}
```

Comment:
```suggestion
    return {param_k: param_v for param_k, param_v in params_dict.items() if param_v}
```
While not mandatory, it is more clear and does the same thing.